.github/actions/unit-tests-action/action.yml

```diff
@@ -59,3 +62,9 @@ runs:
         # we have already installed all dependencies above
         install: false
         cache-key: ${{ runner.os }}-node-${{ hashFiles('**/pnpm-lock.yaml') }}
+
+    - name: Performance test with Lighthouse 🧪
+      shell: bash
```

Comment:
question (non-blocking): why is this necessary?

Reply:
IDE linters require it. I know that this could be removed, but my code will all become highlighted red 🤷‍♂️